.github/workflows/create-release-draft.yml

```diff
@@ -148,43 +158,43 @@ jobs:
             ${{ env.GENESIS_RELEASE }}-genesis-state
             ${{ env.GENESIS_RELEASE }}-genesis-wasm
 
-  ## test again the built docker image ##
```

Comment:
Why we simply remove these testing steps?

Reply:
Hey @BillyWooo I removed them because it was failing and not related to the changes I've made, So that I can only test the changes I've made. I will include them back.